package.json

```diff
@@ -34,6 +34,7 @@
     "tap": "^8.0.0"
   },
   "dependencies": {
+    "lodash": "^4.16.6",
```

Comment:
We can just require `lodash.get` instead of the entire lodash library :)(https://www.npmjs.com/package/lodash.get)